astropy/units/utils.py

```diff
@@ -175,6 +175,20 @@ def sanitize_scale(scale):
         return 1.0
 
     if np.iscomplex(scale):  # scale is complex
+        # Looks like this condition is always false. From np.iscomplex docs:
```

Comment:
Coincidentally, I've been reworking this particular stanza in #7649, so I think it may be best to leave it out here. But I would appreciate if you could have a look at that PR...

Reply:
Got it, I've removed my comment.